skatepark/sass/templates/_header.scss

```diff
@@ -0,0 +1,33 @@
+header {
+	justify-content: space-between; // Apply a cluster (flex?) layout
+	flex-wrap: wrap-reverse;
+	> * { 
+		flex-grow: 1;
+		> * { // Apply a stack layout (page 69 of the every-layout.dev PDF) 
+			margin-top: 20px;
+			margin-bottom: 20px;
+		}
```

Comment:
This is cool, I love the idea of styling the context rather than the element itself.